client/src/components/AuditBoard/BallotAudit.tsx

```diff
@@ -57,7 +59,14 @@ const BallotAudit: React.FC<IProps> = ({
       <div className="ballot-main">
         <SubTitle>Ballot Contests</SubTitle>
         <Formik
-          initialValues={{ interpretations }}
+          initialValues={{
+            interpretations,
+            // Formik only reinitializes if `enableReinitialize` is true and `initialValues`
+            // changes as determined by a deep equality check (reference changes don't trigger
+            // reinitialization). Changing `interpretationsInitializedAt` while leaving
+            // `interpretations` intact accordingly allows consumers to reset the form
+            interpretationsInitializedAt,
+          }}
```

Comment:
I experimented with a few different ideas, and this felt like the simplest and clearest. There are more details on the deep comparison behavior of Formik [here](https://github.com/jaredpalmer/formik/issues/1808). Formik suggests using the `resetForm` prop, but our reset button (i.e. the "Ballot Not Found" button) lives outside the form.

Reply:
I agree given the constraints this seems like a viable and clear solution. Did you experiment at all with moving the Ballot Not Found button inside the form? 

Also just as a note, I've been slowly trying to move away from Formik because it's often got annoying quirks. I've been building all new forms with [react-hook-form](https://react-hook-form.com/). I don't think that will help here but might be worth looking into briefly.

Reply:
I did -- the key challenge was that I'd either have to change the layout of the page, bringing the "Ballot Not Found" down out of the row it's in, _or_ move all of [this content](https://github.com/votingworks/arlo/blob/d154ab646120434cbb04447800c2c2884ab6218a/client/src/components/AuditBoard/Ballot.tsx#L249-L286) nto the `BallotAudit` component and its `form`

Possible! But a little clunky

Reply:
I think it's clearer to include the Ballot Not Found button in the form since it conceptually _is_ submitting the same type of response that the form is submitting ("what's the audit result for this ballot?"). So even though there would be some extra content in there to make the layout work, that seems ok to me.

Reply:
Good point, even if some of the header content feels auxiliary to the form, the "Ballot Not Found" button does feel core to it. Working on that now!